### PR TITLE
Adds login page

### PR DESCRIPTION
**Pull request by zhik**
### Description 
This feature adds a login page for the dashboard view.
image

If access is queried through `QS_VIEW` (widgets) from pages like [RTC's Eviction Crisis Monitor](https://www.righttocounselnyc.org/evictioncrisismonitor) website no password is needed.

---
### Technical Notes

Password is set `REACT_APP_PASSWORD=` in the .env file or Github Secrets 
### Todos

- [ ] Add error if password isn't correct. Currently if the password isn't correct the page just refreshes. 
- [x] Disable login in development env?

### CODE REVIEW

File: contants.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export const VIEW_WIDGET = "widget";
```
Comment (sraby) [inline contants.ts:1]: @zhik did you mean to call this file `constants.ts` rather than "contants?"

**Comment by sraby** [comment]
@zhik this looks great! Thanks so much for your work on this. One idea I had that would address your second TODO is perhaps instead of having a default password of `password`, we could _only_ render the password protection page if that ENV variable is defined. That way, in development, you could just not specify that ENV and it would take you straight to the dashboard without any mention of a password? Totally up to you, though, how you'd like to implement.

File: contants.ts (hunk 1 above)
Comment (zhik) [reply contants.ts:1]: yea :S

**Review by samaratrilling — APPROVED** [review]
Looks good! I agree with Sam's comments

**Pull request merged**